05-emdebian-wchar.diff by Neil Williams <codehelp@debian.org>

Ensure <wchar.h> is available for make_keys via gcc when cross-building
the rest of the package (otherwise it picks up the config.h for the
cross-build).

---
 ncurses/tinfo/comp_hash.c |    1 +
 ncurses/tinfo/make_keys.c |    1 +
 2 files changed, 2 insertions(+)

--- a/ncurses/tinfo/comp_hash.c
+++ b/ncurses/tinfo/comp_hash.c
@@ -39,6 +39,7 @@
  */
 
 #define USE_TERMLIB 1
+#include <wchar.h>
 #include <curses.priv.h>
 
 #include <tic.h>
--- a/ncurses/tinfo/make_keys.c
+++ b/ncurses/tinfo/make_keys.c
@@ -37,6 +37,7 @@
  */
 
 #define USE_TERMLIB 1
+#include <wchar.h>
 #include <curses.priv.h>
 
 MODULE_ID("$Id: make_keys.c,v 1.15 2008/11/16 00:19:59 juergen Exp $")
